### PR TITLE
Added Numpy Page

### PR DESCRIPTION
**Pull request by mmujtabah**
- Added a new page numpy after linear algebra
- Added one problem in numpy topic
- Added favicon icon
- Changed the website title to DeepMLeet
- Improved solution of reshape problem in Linear Algebra

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,15 +1,20 @@
 import streamlit as st
 
+# Set page configuration
+st.set_page_config(page_icon="assets\\favicon.ico", page_title='DeepMLeet')
```
Comment (moe18) [inline app/main.py:4]: I like adding the page_title, however I have a logo I would rather use for this

File: app/NumPy.py
Comment (moe18) [inline app/NumPy.py:1]: I would like the site to be more of a place where people learn machine learning engineering and less programming basics, so I do not think a NumPy page of questions would be needed

**Review by moe18 — COMMENTED** [review]
really liked some of the changes, and had some comments as well, let me know what you think

File: app/main.py (hunk 1 above)
Comment (mmujtabah) [reply app/main.py:4]: I added a template logo so you can replace with your own.

File: app/NumPy.py
Comment (mmujtabah) [reply app/NumPy.py:1]: Got it. We can remove the NumPy page and continue the path of pure machine learning engineering.

File: app/main.py [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
 import MachineLearning
 import DeepLearning
-import about
+import about 
 
 # Define pages in the app
 pages = {
     "About": about,
     "Linear Algebra": LinAlg,
+    "Numpy" : NumPy,
```
Comment (moe18) [inline app/main.py]: remove Numpy from pages

**Review by moe18 — COMMENTED** [review]
looks perfect, one small issue is the Numpy page is still on the main page

File: app/main.py (hunk 2 above)
Comment (mmujtabah) [reply app/main.py]: Sorry forgot to remove it from here.

**Review by moe18 — APPROVED** [review]

**Pull request merged**